src/main/java/org/opentripplanner/raptor/rangeraptor/path/PathParetoSetComparators.java

```diff
@@ -41,37 +42,73 @@ public static <T extends RaptorTripSchedule> ParetoComparator<RaptorPath<T>> par
     boolean includeTimetable,
     boolean preferLateArrival,
     SearchDirection searchDirection,
-    RelaxFunction relaxC1
+    RelaxFunction relaxC1,
+    boolean useC2
   ) {
```

Comment:
This method works but this if-statement tree is messy. Maybe we should think about better solution?